google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -729,4 +730,46 @@ public void testDelete() {
     assertNull(keys.next());
     assertFalse(keys.hasNext());
   }
+
+  @Test
+  public void testRunInTransaction() {
+    Datastore.TransactionCallable<Integer> callable1 =
+        new Datastore.TransactionCallable<Integer>() {
+          private Integer attempts = 0;
+
+          public Integer run(DatastoreReaderWriter transaction) {
+            transaction.get(KEY1);
+            if (attempts < 1) {
```

Comment:
I changed the `attempts` now - as you pointed out, there're 2 `attempts`. I didn't condense the code because I think using "++" in an expression could be error-prone.